ur_rtde_msgs/package.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ur_rtde_msgs</name>
+  <version>0.0.0</version>
+  <description>The ur_rtde_msgs package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="schnell@fzi.de">Tristan Schnell</maintainer>
```

Comment:
Are you going to actively maintain this? Otherwise it might be good to set me as maintainer while keeping you listed as the author.